modules/rosa-deleting-node-tuning.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-tuning-config.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-deleting-node-tuning_{context}"]
+= Deleting node tuning configurations on {hcp-title}
+
+You can delete tuning configurations using the `rosa` CLI.
+
+.Prerequisites
+
+* You have downloaded the latest version of the ROSA CLI.
+* You have a cluster on the latest version .
+* Your cluster has a node tuning configuration that you want delete.
+
+.Procedure
+
+* To delete the tuning configurations, run the following command:
```

Comment:
We should add a warning that deleting a tuning config references in a node pool is not possible. The tuning config needs to be removed first from the ndoe pool then deleted

Reply:
See line 19 for the added note to address this.